modules/log6x-rn-6.0.1.adoc

```diff
@@ -0,0 +1,31 @@
+// Module included in the following assemblies:
+// cluster-logging-release-notes.adoc
+:_mod-docs-content-type: REFERENCE
+[id="cluster-logging-release-notes-6-0-1_{context}"]
+= Logging 6.0.1
+This release includes link:https://access.redhat.com/errata/RHSA-2024:8315[OpenShift Logging Bug Fix Release 6.0.1].
+
+// 4.17
+
+[id="openshift-logging-6-0-1-bug-fixes_{context}"]
+== Bug fixes
+
+* With this update, the default memory limit for the collector has been increased from 1024Mi to 2024Mi. However, users should always adjust their resource limits according to their cluster specifications and needs. (link:https://issues.redhat.com/browse/LOG-6180[LOG-6180])
+
+* Before this update, the Loki Operator failed to add the default `namespace` label to all `AlertingRule` resources, which caused the User-Workload-Monitoring Alertmanager to skip routing these alerts. This update adds the rule namespace as a label to all alerting and recording rules, resolving the issue and restoring proper alert routing in Alertmanager.
+(link:https://issues.redhat.com/browse/LOG-6151[LOG-6151])
+
+* Before this update, the LokiStack ruler component view did not initialize properly, causing an invalid field error when the ruler component was disabled. This update ensures that the component view initializes with an empty value, resolving the issue.
+(link:https://issues.redhat.com/browse/LOG-6129[LOG-6129])
+
+* Before this update, it was possible to set `log_source` in the prune filter, which could lead to inconsistent log data. With this update, the configuration is validated before being applied, and any configuration that includes `log_source` in the prune filter is rejected.
+(link:https://issues.redhat.com/browse/LOG-6202[LOG-6202])
+
+[id="openshift-logging-6-0-1-CVEs_{context}"]
+== CVEs
+
```

Comment:
Please review the CVE's here with errata. I think some are missing here otherwise LGTM.

Reply:
There are no missing CVEs.